runtimevar/awssecretsmanager/example_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awssecretsmanager_test
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"gocloud.dev/runtimevar"
+	"gocloud.dev/runtimevar/awssecretsmanager"
+)
+
+func ExampleOpenVariable() {
+	// Establish an AWS session.
+	// See https://docs.aws.amazon.com/sdk-for-go/api/aws/session/ for more info.
+	sess, err := session.NewSession(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Construct a *runtimevar.Variable that watches the variable.
```

Comment:
Maybe add a comment here as well about what cfg-variable-name could be, and/or make the example more realistic.